.travis.yml

```diff
@@ -14,6 +14,7 @@ install:
   # workaround until https://github.com/ansible/molecule/issues/1727 is fixed
   - pip uninstall -y testinfra
   - pip install testinfra
+  - pip install jmespath
```

Comment:
is there an rpm based package that provides this functionality?  We can't use pip installed packages in our products.

Reply:
This "pip install jmespath" addition is just to make the travis CI test back to work.  The other CI tests are just fine (except rhel/centos-6 :) without it.  Should we still not use pip in .travis.yml?  I'm not sure how the CI test hosts are configured, but it looks to me jmespath is missing only in the host for the travis CI and it is available in the other test hosts...  Which is the valid initial status?

In terms of  the rpm packages, there are 2 of them:
```
python2-jmespath.noarch : JSON Matching Expressions
python3-jmespath.noarch : JSON Matching Expressions
```

Reply:
@richm yes, there is a jmespath package and the rhel-system-roles package requires it in order to work - it is properly expressed as an RPM dependency. This is not related to Travis though - it does not even use a RPM distribution.